src/nph.nim

```diff
@@ -10,7 +10,18 @@ import
 
 import "$nim"/compiler/idents
 
-import std/[parseopt, strutils, os, sequtils]
+import std/[parseopt, strutils, os, sequtils, terminal]
+import pkg/hldiffpkg/edits
+import pkg/adix/lptabz
+
+# Silence LPTabz warnings by redirecting to /dev/null
+when defined(posix):
+  lpWarn = open("/dev/null", fmWrite)
+elif defined(windows):
+  lpWarn = open("NUL", fmWrite)
+else:
+  # Fallback: just use stderr but set max warnings to 0
+  lpMaxWarn = 0
```

Comment:
why not use this for all platforms? it avoids opening a device which may or may not be available.